DirectEmployers/MyJobs
# PD-3455 analytics print weekly monthly charts not working properly

## What's this PR do?
* Shows the correct Timeline chart in print view for the timespan selected  (formerly it would always show `Day` even when `Week` or `Month` were selected.
* Persists the timespan across tabs and dimensions
* The timeline state has been isolated under `state.main.timeline`

## Where should the reviewer start?
1. `initial-state.js`
2. `main-reducer.js` -> `SET_TIMELINE_TIMESPAN`
3. `tab-actions.js`
4. `TabsContainer.js`
5. `ChartContainer.js`
6. `TimelineChart.js`

## How should this be manually tested?
1. Go to Analytics 
2. Show Timeline 
3. Change Timespan (to `Week` or `Month`)
4. Click Print Report button and select chart
5. Click Print
6. The correct Timeline should be visible in the print preview
